themes/Hacks2013/style.css

```diff
@@ -36,7 +36,7 @@ abbr.updated, abbr.published { border: 0; }
 
 /* =General elements *********/
 html { background: #454545; }
-body { position: relative; color: #fac817; font: 10px/1.6 Helvetica, "Helvetica Neue", Arial, sans-serif; }
+body { position: relative; color: #fac817; font: 10px/1.6 Open Sans, "Helvetica Neue", Arial, sans-serif; }
```

Comment:
Multi-word font names need to be in quotation marks: <code>"Open Sans",</code> (make sure the comma is outside the quote).

And since you haven't included Open Sans in <code>@font-face</code> rule this will rely on getting it from Tabzilla. That's fine, I'm just making note of the dependency. If Tabzilla is removed or changes, we'll lose Open Sans as well.